evap/staff/importers.py

```diff
@@ -70,7 +70,7 @@ def validate(self):
 
 
 @dataclass
-class EvaluationData:
+class EvaluationData:  # pylint: disable=too-many-instance-attributes
```

Comment:
Do you think this is valid? I dont know how to reduce the number of attributes and one could argue that this is a data class

Reply:
I think this is fine